### DISCUSSION TITLE
I haven’t received my education benefits.

### DISCUSSION THREAD

**Discussion by Kevinye0116** [community/community#134579] [GitHub Education]
### Hi everyone,

I haven’t received my education benefits which has been approved. Can the team give me some help?

**Comment by DTCarlox77** [comment]
¿Ya te has registrado con un correo edu? Eso es lo principal, además, para solicitar github pro necesitarás haber aplicado un sistema de verificación de dos pasos con una aplicación de autenticación, puedes usar la de Authenticator de Microsoft. Luego ve a https://github.com/education/students te pedirán varias verificaciones para definir si eres o no un estudiante activo. Espero haber ayudado con esto.

**Reply by Kevinye0116** [reply]
I’ve done it and I’ve got the approval.

**Comment by Sunjid-Ahmed** [comment]
response #1

**Comment by davevad93** [comment]
Hi @Kevinye0116 ,

What's the state of your application? If it says "``Congrats! Your request for @Kevinye0116 was approved and your benefits will be available soon.``" it means you've been approved but you still have to wait in order to use your benefits. 

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Kevinye0116 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by elsycharles** [comment]
Hi @Kevinye0116 
Before you apply you have to do all the Github Education Application
[https://education.github.com/discount_requests/application](https://education.github.com/discount_requests/application)
![Capture d’écran du 2024-08-04 16-51-31](https://github.com/user-attachments/assets/c3b985c1-2cf6-43e1-9218-c27494a680f8)
